yarn.lock

```diff
@@ -1669,7 +1669,6 @@ antigravity@artsy/antigravity:
   version "0.1.3"
   resolved "https://codeload.github.com/artsy/antigravity/tar.gz/a4438d2fe9d0cdf71f1c47faba371cd3d004e140"
   dependencies:
-    coffeescript "1.11.1"
```

Comment:
odd

Reply:
Yeah, not sure what's up w/ this, but it shouldn't matter.